syntax = "proto3";

package aom.iot.action;

message Action {
    string name = 1;
    string uuid = 2;
    string user_uuid = 3;
    string trigger_topic = 4;
    uint32 trigger_state = 5;
    uint32 response = 6;
}
